bundle/bin/rke2-uninstall.sh

```diff
@@ -47,6 +47,16 @@ uninstall_remove_files()
     rm -rf /etc/rancher/rke2
     rm -rf /var/lib/kubelet
     rm -rf /var/lib/rancher/rke2
+    remove_directory_if_empty "/var/lib/rancher"
+}
+
+remove_directory_if_empty()
+{
+    if [ -z "$(ls -A ${1})" ]; then
+        rmdir ${1}
+    else
+        echo "Skip remove directory: ${1} is not empty."
+    fi
```

Comment:
```suggestion
    rm -d /var/lib/rancher
```

Reply:
Also, if we are doing this, should we not also do the same for `/etc/rancher` ?

Reply:
We may need to check with @ShylajaDevadiga and @rancher-max in case this breaks any automation on their side.

Reply:
Our automation brings up new instances each time so this should be okay for us! Thank you for checking

Reply:
Thanks @dweomer ! I wasn't sure if the /etc/rancher/node/password was kept on purpose. Addressed.

I'm a little concerned about using `rm -d`. When used, the function will exit on the first fail. That means if `/etc/rancher` is not empty and `/var/lib/rancher` is empty. Both directories will be kept, and all the following commends in that function won't be processed.

Reply:
you can `|| true` or just `set +e` beforehand and then `set -e` back again afterwards.

Reply:
@brandond thanks!

Reply:
> Our automation brings up new instances each time so this should be okay for us! Thank you for checking

Cool. Just wanted to make sure since the expectation was that this is removed and this changes it. I know @ShylajaDevadiga had a list of files and directories to check for in the past.